internal/corazawaf/transaction.go

```diff
@@ -1576,6 +1576,14 @@ func (tx *Transaction) String() string {
 	return res.String()
 }
 
+func (tx *Transaction) UnixTimestamp() int64 {
```

Comment:
This is controversial and that is why I'd like to use use cases before exposing this. Why not a time.Time? Not saying that is the answer, just saying we need concrete use cases.